.env.example

```diff
@@ -0,0 +1,6 @@
+FDR_FASE1_BASE_URL="https://api.dev.platform.pagopa.it/fdr-legacy/service-internal/v1"
+FDR_FASE1_API_KEY="insert-key-here"
+BLOB_STORAGE_FDR3_CONTAINER="insert-blob-storage-container-here"
+FDR_SA_CONNECTION_STRING="blob-storage-connection-string-here"
+TABLE_STORAGE_CONN_STRING="table-storage-connection-string-here"
+ERROR_TABLE_NAME="insert-table-name-here"
```

Comment:
insert the table name

Reply:
Yes I can but it's only an example without values